### DISCUSSION TITLE
想问一下如何设置取消参考文献中作者名只显示3个的限制

### DISCUSSION THREAD

**Discussion by RPKU** [sjtug/SJTUThesis#1016] [提问] [answered]
图书馆反馈在参考文献列表里，超过3位作者时，所有作者也需全部列出

**Comment by hushidong** [comment, marked as answer]
biblatex增加选项maxbibnames=99


即在setup.TEX中，修改：

\usepackage[backend=biber,style=gb7714-2015,maxbibnames=99]{biblatex}
